quiche/quic/core/quic_connection.cc

```diff
@@ -3485,10 +3485,10 @@ bool QuicConnection::WritePacket(SerializedPacket* packet) {
       result = SendPacketToWriter(
           packet->encrypted_buffer, encrypted_length, send_from_address.host(),
           send_to_address, writer_, GetEcnCodepointToSend(send_to_address));
-      // This is a work around for an issue with linux UDP GSO batch writers.
+      // This is a workaround for an issue with linux UDP GSO batch writers.
```

Comment:
the `a` (article) here means that the desired item is `workaround` (n) not `work around` (vp).